.github/workflows/branch_pages_deploy.yml

```diff
@@ -17,38 +17,71 @@ jobs:
         with:
           node-version-file: '.nvmrc'
 
-      - name: 'Set BRANCH_NAME variable'
```

Comment:
The `GITHUB_REF` is incorrect since the `workflow_dispatch` doesn't checkout the forked branches. 
The PR data including the `branch` is now retrieved with a call to the GH API.